projects/public-search/src/app/document-detail/request/request.component.html

```diff
@@ -17,45 +17,31 @@
 @if (patron) {
   @if (!requestDialog) {
     @if (canRequest && canRequest.can) {
-      <dl id="record-request-{{ record.metadata.pid }}" class="row" [ngClass]="{'row pl-4': recordType === 'item', 'mb-3': recordType === 'holding'}">
-        @if (recordType !== 'holding') {
-          <dt class="mt-2 mb-3 col-lg-2 col-sm-3">&nbsp;</dt>
+      <p-button
```

Comment:
[nice to have enhancement] Add an icon 'pi pi-cart-arrow-down' to this button

Reply:
Can I put the outlined for all type of requests?

Reply:
Yes